### DISCUSSION TITLE
Add collections reordering in the collections list

### DISCUSSION THREAD

**Discussion by nelolka** [directus/directus#3530] [Feature Requests]
The collections in the collections list are ordered alphabetically now. It would be great if you add the sort feature for the collections.

**Comment by benhaynes** [comment]
And "native" grouping. You can already create groups with the _override_, but it would be nice to support a GUI for all of this. Ideally, this order/grouping in the override can still be dynamic... meaning that if new collections are added, they still have the option to show up in an "overflow" group.

**Reply by sercanvirlan** [reply]
Hey ben!

What do you mean by "create groups with the override" is this action possible somehow ?

**Reply by sercanvirlan** [reply]
wow i found where i can do, it's little complicated, adding for guidance someone else like me 

Go Roles & Permiossions
Choose one and scroll down

![Screen Shot 2021-02-18 at 16 28 22](https://user-images.githubusercontent.com/533209/108363783-5bc5ec00-7206-11eb-9963-babe25ffd516.jpg)

**Reply by benhaynes** [reply]
Hmm, could you explain which part is complicated/confusing? It's only asking for three things:

* Group Name — optional, if you want to group collections
* Type — if the group starts open/closed
* Collections — just a dropdown of collections to select/order

![Screen Shot 2021-02-18 at 10 18 51 AM](https://user-images.githubusercontent.com/522079/108378154-bfcebd00-71d2-11eb-8ed7-c78502cb49f1.png)

**Reply by sercanvirlan** [reply]
Actually complicated part is finding this where to do, maybe a note would be great under presets & bookmarks settings. Anyway i'm pleasured for the UX thanks! 🙏🏻

**Reply by t7tran** [reply]
This Collection Navigation feature will be super useful if `Group Name` supports translations. And @benhaynes, I was under the impression that `Group Name` is required (see the asterisk), but indeed it is optional.

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
